deploy/coins/ethereum/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Prepare Environment
+FROM alpine:latest as builder
+
+ENV GETH_VERSION=1.10.23
+
+WORKDIR /home/ethereum
+
+RUN apk update \
+	&& apk --no-cache add ca-certificates gnupg bash su-exec
+
+# Get Binary
+RUN wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-${GETH_VERSION}-d901d853.tar.gz\
+	&& wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-${GETH_VERSION}-d901d853.tar.gz.asc
+
+# Verify Binary
+# refer to https://geth.ethereum.org/downloads/#openpgp_signatures
+# for the PGP keys of builders and developers
+ENV KEYS F9585DE6 C2FF8BBF 9BA28146 7B9E2481 D2A67EAC E058A81C 05A5DDF0 1CCB7DD2
```

Comment:
9BA28146 - Linux build
E058A81C - Felix Lange
05A5DDF0 - Martin Holst Swende
1CCB7DD2 - Péter Szilágyi

NACK to including random build servers as well, unless they are co-signing for some reason.

Also, what's the security of 4 byte fingerprints? What happens if the Ubuntu key-server has two keys with this fingerprint? Will it error due to ambiguity?